tests/unit/editors/Bugs/bugs-test.js

```diff
@@ -0,0 +1,1620 @@
+$(document).ready(function () {
```

Comment:
The ready function is not really required for these tests anymore so you can remove it :)